ui/src/table.tsx

```diff
@@ -930,7 +931,15 @@ export const
       }
       // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [])
-    React.useEffect(() => setFilteredItems(items), [items])
+
+    React.useEffect(() => {
```

Comment:
This seems like a common thing we use and will use in the future due to React being what it is. Wdyt about extracting this into a separate custom hook? Something like:

```tsx
import React from 'react'

const useUpdateOnlyEffect = (func: (...args: any) => any, deps: any[]) => {
  const didMount = React.useRef(false)
  React.useEffect(() => { didMount.current ? func() : didMount.current = true }, deps)
}

export default useUpdateOnlyEffect
```

Reply:
Yes, you are right. We are using it quite often so why not. :)